### ISSUE TITLE
[dind] SDN_NODE=true -> "-o option doesn't exist"

### ISSUE THREAD

**Issue by jayunit100** [openshift/origin#6561]
Running openshift's dind starter I ran into the following issue on Fedora 21 (locally **without vagrant vm**, I get some kind of minor `-o` doesnt exist error.

```
++ NETWORK_PLUGIN=redhat/openshift-ovs-subnet                                                                                  [96/1840]
++ getopts :i:n:c:fs opt
++ case $opt in
++ echo 'Invalid option: -o'
Invalid option: -o
++ exit 1
```

I think just adding "o" to getopts in `provision-config.sh` fixes this `+while getopts ":i:n:c:fso" opt; do` ?

@marun

**Comment by jayunit100** [comment]
To be clear, I discovered this only by setting the `SDN_NODE=true` and running `hack/dind-cluster.sh start` locally on a fedora box.

**Comment by jayunit100** [comment]
(unrelated to the core issue) And, while I have your attention, what certs should I use when running the e2es from dind? 

```
 sudo _output/local/bin/linux/amd64/e2e.test --host="https://172.17.0.38:8443" --provider="local" --ginkgo.v=true --ginkgo.focus=Networking --kubeconfig=./test/old-start-configs/v1.0.0/config/openshift.local.config/master/openshift-master.kubeconfig --repo-root=./
```

Seems not to work (INFO: Error getting replication controllers in namespace 'kube-system': Get https://172.17.0.38:8443/api/v1/namespaces/kube-system/replicationcontrollers?timeout=30s: x509: certificate signed by unknown authority)

**Comment by marun** [comment]
@jayunit100 by certs I assume you mean what kubeconfig to use?  dind drops an rc (dind-openshift.rc) that you can source (. dind-openshift.rc) to set KUBECONFIG.  By default the value will be /tmp/openshift-dind-cluster/openshift/openshift.local.config/master/admin.kubeconfig.

Also, #6559 will resolve this issue by making SDN_NODE=true the default for openshift-sdn.

**Comment by jayunit100** [comment]
okay.  Closed in favor of #6559 lets merge that soon.
re the certs, i found that the ones created in /tmp work.
